src/utils/helpers.ts

```diff
@@ -1,93 +1,92 @@
-import { DEFAULT_CHAIN_ID, DefaultPriceConfig } from "../configs";
```

Comment:
If `DefaultPriceConfig` is not used anywhere then we are free to delete this file